### PR TITLE
fix CloudService SDK bug due to capitalization

### PR DESCRIPTION
**Pull request by grizzlytheodore**
# Choose a PR Template

addressing 
https://github.com/Azure/azure-sdk-for-net/issues/26713
https://github.com/Azure/azure-sdk-for-net/pull/28922

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

### CODE REVIEW

File: specification/compute/resource-manager/Microsoft.Compute/CloudserviceRP/stable/2022-04-04/cloudService.json [hunk 1]
```diff
@@ -2090,19 +2090,19 @@
       "description": "Describes the properties of the load balancer configuration.",
       "type": "object",
       "properties": {
-        "frontendIPConfigurations": {
+        "frontendIpConfigurations": {
           "description": "Specifies the frontend IP to be used for the load balancer. Only IPv4 frontend IP address is supported. Each load balancer configuration must have exactly one frontend IP configuration.",
           "type": "array",
           "items": {
-            "$ref": "#/definitions/LoadBalancerFrontendIPConfiguration"
+            "$ref": "#/definitions/LoadBalancerFrontendIpConfiguration"
```
Comment (ArcturusZhang) [inline specification/compute/resource-manager/Microsoft.Compute/CloudserviceRP/stable/2022-04-04/cloudService.json:2097]: You actually do not need to change this, this change breaks the references

Comment (grizzlytheodore) [reply specification/compute/resource-manager/Microsoft.Compute/CloudserviceRP/stable/2022-04-04/cloudService.json:2097]: so can i close this PR?

Comment (grizzlytheodore) [reply specification/compute/resource-manager/Microsoft.Compute/CloudserviceRP/stable/2022-04-04/cloudService.json:2097]: @ArcturusZhang

**Comment by grizzlytheodore** [comment]
@ArcturusZhang  could you comment

**Pull request closed without merging**